packages/react-pdf-toolkit/src/components/icons/AlertIcon/AlertIcon.tsx

```diff
@@ -1,4 +1,12 @@
 import { Image } from '@/components/atoms/Image';
-import alertIcon from '../../../assets/pdf-alert-icon.png';
+import { FunctionComponent } from 'react';
+import alertGreyedIcon from '../../../assets/alert-greyed-icon.png';
+import alertIcon from '../../../assets/alert-icon.png';
 
-export const AlertIcon = () => <Image width={16} height={16} src={alertIcon} />;
+export interface AlertIconProps {
```

Comment:
Is this type exported to the consumers of the package?

Reply:
P2

Reply:
Not directly type but component is used by consumers. To avoid building issues with missing types exporting type as well.